redaxo/src/core/fragments/core/top.php

```diff
@@ -12,7 +12,8 @@
         foreach ($files as $file) {
             $path = rex_path::base(rex_path::absolute($file));
             if ($mtime = @filemtime($path)) {
-                $file .= '?buster='. $mtime;
+                // use a frontend-api-function to make sure we dont have to wait for session locks.
+                $file = rex_url::frontendController(['asset' => $file, 'buster' => $mtime] + rex_api_asset_stream::getUrlParams());
```

Comment:
dadurch dass hier jetzt das css file aus einem anderen pfad geladen wird passen die pfade der referenzierten fonts im css nicht mehr.. müsste man dann in diesem zuge anpassen.